python/graphstorm/model/utils.py

```diff
@@ -87,37 +88,73 @@ def save_model_results_json(conf, test_model_performance, save_perf_results_path
         with open(json_path, 'w', encoding='utf-8') as f:
             json.dump(model_results_and_conf, f, ensure_ascii=False, indent=4)
 
-def save_sparse_embeds(model_path, embed_layer):
+def _get_sparse_emb_range(num_embs, local_rank, world_size):
```

Comment:
we should have a unit test to test this function.

Reply:
Added